includes/notes/class-wc-payments-notes-additional-payment-methods.php

```diff
@@ -1,139 +0,0 @@
-<?php
-/**
- * Display a notice to merchants that have WCPay installed (and don't have UPE enabled)
- * to inform them that they can enable additional payment methods.
```

Comment:
In this implementation the methods are listed in the settings from the beginning but in disabled state. Shouldn't we still let the merchant's know that they can enable some additional payment methods?